testsuite/config/cucumber.yml

```diff
@@ -3,7 +3,7 @@ action_chains: --tags @scope_action_chains
 ansible: --tags @scope_ansible
 api: --tags @scope_api
 building_container_images: --tags @scope_building_container_images
-centos: --tags @centos_minion
+centos: --tags @rhlike_minion
```

Comment:
I don't see the usage of this scope in https://github.com/SUSE/susemanager-ci/pull/595/files. I see @scope_res could we have been using this wrong all along? Please verify this because you are not using @rhlike_minion anywhere now as well.

Reply:
Good catch :smile: . I caught it too. I was tempted to remove it, but there might be this reason for this weirdness:

There is no `scope_centos` defined. They probably did not do it because it would have been an exact duplicate of `@centos_minion` (not in the semantics, but for the places where it appears).

If the user wants to toggle the CentOS scope on/off, that's the only way to acheive it.

It's only an educated guess. Maybe Jordi can confirm.

Reply:
I don't think I see the mapping of the scope to the centos_minion tag, which are different things in scope. Maybe @jordimassaguerpla can take a look at this?